specification/storage/data-plane/Microsoft.FileStorage/stable/2024-11-04/file.json

```diff
@@ -6543,6 +6546,17 @@
         "permission": {
           "type": "string",
           "description": "The permission in the Security Descriptor Definition Language (SDDL)."
+        },
+        "format": {
+          "type": "string",
+          "enum": [
+            "SDDL",
+            "binary"
+          ],
+          "x-ms-enum": {
+            "name": "FilePermissionKeyFormat",
+            "modelAsString": false
```

Comment:
If you use modelAsString:false, then the set of enum values can NEVER change. Is this what you want?

Reply:
What can't we add additional enum values in the future?

Reply:
No, if you want the flexibility to add more in the future, then use modelAsString:true (not false).